.github/workflows/run_ert2_test_data_setups.yml

```diff
@@ -0,0 +1,50 @@
+name: Run test-data
+
+on: [push, pull_request]
+
+env:
+  ERT_SHOW_BACKTRACE: 1
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.6]
```

Comment:
The order of the versions is itching me. My apologies, but can you please fix it 😂

Reply:
Not a fan of appending? ;) Fixed!